recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,44 @@
 c_stdlib_version:              # [linux]
   - 2.17                       # [linux]
+
+c_stdlib_version:                                   # [linux64]
+  - 2.17                                            # [linux64]
+  - 2.17                                            # [linux64]
+  - 2.17                                            # [linux64]
+  - 2.28                                            # [linux64]
+docker_image:                                       # [linux64]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64      # [linux64]
+  - quay.io/condaforge/linux-anvil-cuda:11.8        # [linux64]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64      # [linux64]
+  - quay.io/condaforge/linux-anvil-alma-x86_64:8    # [linux64]
+# rest just to make smithy happy
+c_compiler_version:                                 # [linux64]
+  - 13                                              # [linux64]
+  - 11                                              # [linux64]
+  - 12                                              # [linux64]
+  - 12                                              # [linux64]
+cxx_compiler_version:                               # [linux64]
+  - 13                                              # [linux64]
+  - 11                                              # [linux64]
+  - 12                                              # [linux64]
+  - 12                                              # [linux64]
+fortran_compiler_version:                           # [linux64]
+  - 13                                              # [linux64]
+  - 11                                              # [linux64]
+  - 12                                              # [linux64]
+  - 12                                              # [linux64]
+cdt_name:                                           # [linux64]
+  - cos7                                            # [linux64]
+  - cos7                                            # [linux64]
+  - cos7                                            # [linux64]
+  - cos7                                            # [linux64]
+cuda_compiler:                                      # [linux64]
+  - None                                            # [linux64]
+  - nvcc                                            # [linux64]
+  - cuda-nvcc                                       # [linux64]
+  - cuda-nvcc                                       # [linux64]
+cuda_compiler_version:                              # [linux64]
+  - None                                            # [linux64]
+  - 11.8                                            # [linux64]
+  - 12.0                                            # [linux64]
+  - 12.6                                            # [linux64]
```

Comment:
Should the CUDA versions be set manually per feedstock rather than in the global pinnings? I'm reading the thread at https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/6630, and it seems like there's no consensus yet on whether to have 3 versions (11.8, 12.0, 12.6) or just 2 versions (11.8, 12.6).

Reply:
I have a slight preference to rely on conda-forge's global-pinnings so that we don't need to unpin things in this `recipe/conda_build_config.yaml` file in the future (e.g. when a newer CUDA version comes out). But no strong opinions either on adding CUDA 12.6 here, especially if any users require it right now (unsure if CUDA 12.6 unlocks any new functionality in pytorch3d). Any thoughts @Tobias-Fischer?